src/app/constants/game-constants.js

```diff
@@ -26,6 +26,15 @@ export default {
         UI: 'ui',
     },
 
+    CREATURE_ANIM_STATS:{
+        speed:200,
```

Comment:
Indentation :D

Reply:
Yea, I think our Editors use different styling rules, I'm used to 2 spaces indentation lol

Reply:
I like 2 spaces as well, I will try to keep an eye out. Mine seems to want to do 4 sometimes.